core/Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "Ribasim"
 uuid = "aac5e3d9-0b8f-4d4f-8241-b1a7a9632635"
-authors = ["Deltares and contributors"]
+authors = ["Deltares and contributors <ribasim.info@deltares.nl>"]
```

Comment:
Maybe add the email address to the Python packages as well?

Reply:
Like ada5bd384c754eb8f4b9c92d2256595dc8bc4fd1? This removes individual authors. Much easier to keep up to date though, it was already out of date. If we want to highlight individual contributions that would probably best be done in one place.